core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -637,6 +638,7 @@ Builder populate() {
       defineMethod(EXP, BuiltInMethod.EXP.method, NullPolicy.STRICT);
       defineMethod(POWER, BuiltInMethod.POWER.method, NullPolicy.STRICT);
       defineMethod(ABS, BuiltInMethod.ABS.method, NullPolicy.STRICT);
+      defineMethod(LOG2, BuiltInMethod.LOG2.method, NullPolicy.STRICT);
```

Comment:
Should this use the LogImplementor defined below?

Reply:
I tried it, but apparently it's not a good idea because mysql and postgres handle the log10(0) function inconsistently

Reply:
I don't understand what you mean. For each function you have to return the same results as the corresponding function in MySQL.

Reply:
yes, so I didn't put it in the LogImplementor defined below

Reply:
If I put it in LogImplementor, I might have to do more work than now, because the log function in postgres and mysql obviously handles 0 differently, so I raised jira on this issue.